docs/docsite/rst/playbook_guide/playbook_pathing.rst

```diff
@@ -21,25 +21,35 @@ Task paths
 
 Relative paths used in a task typically refer to remote files and directories on the managed nodes. However, paths passed to lookup plugins and some paths used in action plugins such as the "src" path for the :ansplugin:`template <ansible.builtin.template#module>` and :ansplugin:`copy <ansible.builtin.copy#module>` modules refer to local files and directories on the control node.
 
-Resolving local relative paths
-------------------------------
+How plugins resolve local relative paths
```

Comment:
most of the time plugins do not resolve the paths, but the core engine does, some plugins might have path resolution but then it can be different than what is described here.

Reply:
Thanks @bcoca I've reverted this to the original title.